docs/style.md

```diff
@@ -136,42 +154,46 @@ end
 If the profile is pointed to a remote endpoint using the `command` resource
 will run it on the remote OS.
 
-**Avoid Ruby IO on files**
-
-Avoid: `File.new("filename").read`
-Avoid: `File.read("filename")`
-Avoid: `IO.read("filename")`
-Use: `file("filename")`
+## Avoid Ruby IO on files
 
-Similar to the command interactions these files will only be read localy
-with Ruby's internal calls. If you run this test against a remote target it won't
-read the file from the remote endpoint, but from the local OS instead.
-Use the `file` resource to read files on the target system.
+Similar to the command interactions these files will only be read locally with
+Ruby's internal calls. If you run this test against a remote target it won't
+read the file from the remote endpoint, but from the local OS instead. Use the
+`file` resource to read files on the target system.
 
-In general, try to avoid Ruby's IO calls from within InSpec controls and
-use InSpec resources instead.
+Avoid: `File.new("filename").read`</br>
+Avoid: `File.read("filename")`</br>
+Avoid: `IO.read("filename")`</br>
+Use: `file("filename")`
 
+In general, try to avoid Ruby's IO calls from within InSpec controls and use
+InSpec resources instead.
 
-### 7. Avoid Ruby gem dependencies in controls
+## Avoid Ruby gem dependencies in controls
 
-In addition to avoiding system-level gems and modules you should also limit
-the use of external dependencies to resource packs or plugins. Gems need to be
+In addition to avoiding system-level gems and modules you should also limit the
+use of external dependencies to resource packs or plugins. Gems need to be
 resolved, installed, vendored, and protected from conflicts. We aim to avoid
-exposing this complexity to users of InSpec, to make it a great tool even if
-you are not a developer.
-
-Developers may still use external gem dependencies but should vendor it
-with their plugins or resource packs.
+exposing this complexity to users of InSpec, to make it a great tool even if you
+are not a developer.
 
+Developers may still use external gem dependencies but should vendor it with
```

Comment:
I think this should be rephrased, as plugins and resource packs treat gems differently.

Perhaps:

Plugins should declare gem dependencies in their gemspec, and then rely on the plugin installation facility to install and manage dependencies.

I'm not sure how to ship gems with resource packs.  I don't think vendoring them will work (how would that load path be set?)

Feel free to address that on a separate PR.

Reply:
I agree it should be fixed. I'll add your suggestion to this PR. I'm also not sure about vendoring gems in resource packs.